feature: add next.js, react quickstarts (#1093)
knocklabs/docs: content/getting-started/quick-start/general.mdx

---
title: Get started with Knock
description: Quickly get up and running with Knock.
tags: ["getting started"]
section: Getting started
---

> In this guide, you'll integrate Knock with your backend web application and send your first notification using Knock.

Reviewer: ```suggestion
This page covers how to integrate Knock with your backend web application and send your first notification using Knock.
```

        You do this using the `Commit` button  in the header of the workflow canvas. You can review your changes using a diff viewer and leave a meaningful commit message about what has changed.
        <img className="rounded-md mx-auto border border-gray-200" alt="diff your changes and leave a message" style={{marginBottom: "12px"}} src="/images/quick-start/commit-diff.png"/>
      </Accordion>
    </AccordionGroup>
    Now we're ready to trigger our workflow via the Knock API. You can also learn more about workflows and channels in Knock via our [guide on designing workflows](/send-notifications/designing-workflows).

  </Step>
  <Step title="Trigger your workflow">
    Now, you'll trigger your workflow to notify a set of users. When triggering workflows, you need to provide the following required pieces of data in your call to the Knock API:

    - `recipients` – The list of users to notify.
    - `data` – The variable data that will populate your notification templates. Here you'll use the sample data payload we grabbed in the previous step.

>     In the example below, we trigger a new comment notification workflow for two project members, using [inline identification](/api-reference/workflows/trigger).

Reviewer 2: Unsure if it's more helpful to have this link to the identify recipient docs?

```suggestion
    In the example below, we trigger a new comment notification workflow for two project members, using [inline identification](/managing-recipients/identifying-recipients#inline-identification).
```
